tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -44,10 +44,16 @@ def test_dcv_configuration(
     # dcv connect show url
     env = operating_system.environ.copy()
     env["AWS_DEFAULT_REGION"] = region
+    operating_system.system(
+        r"sudo sed -i 's/\#   StrictHostKeyChecking ask/   StrictHostKeyChecking no/g' /etc/ssh/ssh_config"
+    )
     result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
     assert_that(result.stdout).matches(
         r"Please use the following one-time URL in your browser within 30 seconds:\n"
-        r"https:\/\/(\b(?:\d{1,3}\.){3}\d{1,3}\b):" + str(dcv_authenticator_port) + r"\?authToken=(.*)"
+        r"https:\/\/(\b(?:\d{1,3}\.){3}\d{1,3}\b):" + str(dcv_port) + r"\?authToken=(.*)"
+    )
+    operating_system.system(
```

Comment:
if there is a failure in the assertions or in the run command this config is not going to be reverted. We should make this more robust (e.g. try/finally or use fixtures)